db/schema.rb

```diff
@@ -1,45 +0,0 @@
-# This file is auto-generated from the current state of the database. Instead
```

Comment:
Should we be deleting this schema? I was very confused when you asked me about it earlier but this is the right one so I think we need to keep it.